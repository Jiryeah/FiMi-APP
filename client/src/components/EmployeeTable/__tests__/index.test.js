import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Table from '..';

afterEach(cleanup);

describe('Table Component', () => {
  // baseline Table test
  it('renders', () => {
    render(<Table />);
  });

  it('matches snapshot DOM node structure', () => {
    // render Table snapshot
    const { asFragment } = render(<Table />);
    expect(asFragment()).toMatchSnapshot();
  });
});
