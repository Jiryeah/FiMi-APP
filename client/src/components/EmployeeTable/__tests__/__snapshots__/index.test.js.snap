// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    Table
  </div>
</DocumentFragment>
`;
